README.md

```diff
@@ -87,6 +87,16 @@ AI: [real_time/surface_src](real_time/surface_src)
 
 IO Controller: [real_time/arduino_src/radio_buggy_mega](real_time/arduino_src/radio_buggy_mega)
 
+## File Structure 
+
+THe software system is seperated into two main sections:
+* offline -> where code which is not executed in realtime on the buggy is stored
+* realTime -> where code which is being run in realtime on the buggy is stored 
+
+Theis folder also contains: 
+* Pictures for this readmde file
+* configuration files
+
 ## Sponsors
 
 Special thanks to Microsoft for providing Alice a host interface on a Surface Pro 3!
```

Comment:
well not yet at least

On Mon, Sep 14, 2015 at 7:29 PM, Trevor Decker <mr.trevor.decker@gmail.com>
wrote:

> No thats a typo
>
> On Mon, Sep 14, 2015 at 7:20 PM, Matthew Sebek <notifications@github.com>
> wrote:
>
>> In README.md
>> <https://github.com/CMU-Robotics-Club/RoboBuggy/pull/173#discussion_r39459297>
>> :
>>
>> > @@ -87,6 +87,16 @@ AI: [real_time/surface_src](real_time/surface_src)
>> >
>> >  IO Controller: [real_time/arduino_src/radio_buggy_mega](real_time/arduino_src/radio_buggy_mega)
>> >
>> > +## File Structure
>> > +
>> > +THe software system is seperated into two main sections:
>> > +* offline -> where code which is not executed in realtime on the buggy is stored
>> > +* realTime -> where code which is being run in realtime on the buggy is stored
>> > +
>> > +Theis folder also contains:
>> > +* Pictures for this readmde file
>> > +* configuration files
>> > +
>> >  ## Sponsors
>> >
>> >  Special thanks to Microsoft for providing Alice a host interface on a Surface Pro 3!
>>
>> Surface Pro 2 for 2014; did we get a new present?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/CMU-Robotics-Club/RoboBuggy/pull/173/files#r39459297>
>> .
>>
>
>
>
> --
> Sincerely,
>
> Trevor Decker
>



-- 
Sincerely,

Trevor Decker